lib/src/impl_stub/impl_stub.hmac.dart

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+part of 'impl_stub.dart';
+
+final class _StaticHmacSecretKeyImpl implements StaticHmacSecretKeyImpl {
+  const _StaticHmacSecretKeyImpl();
+
+  @override
+  Future<HmacSecretKeyImpl> importRawKey(List<int> keyData, Hash hash, {int? length}) {
+    throw UnimplementedError();
```

Comment:
Can we throw `throw UnimplementedError('Not implemented');`

Or even just make a `final _notImplemented = UnimplementedError('Not implemented');` inside `impl_stub.dart`, then we can use that in all the stubs.

Perhaps this is a follow up PR :D

Reply:
Updated to `throw UnimplementedError('Not implemented');` as is in other files. Missed it here.

> Or even just make a final _notImplemented = UnimplementedError('Not implemented'); inside impl_stub.dart, then we can use that in all the stubs.

Will update to this structure in a follow up PR